### PR TITLE
Mbarba/addition test

### PR DESCRIPTION
**Pull request by MatBarba**
Create a test for the addition prepare similar to the genome prepare pipeline, using a highly reduced genbank file as cache.

Also create a specific test Nextflow config file to be used only when testing (this is to replace the labels that require a lot of resources, which are not needed for a test). The tests can run fully locally this way.

Add how to run those tests in the README

### CODE REVIEW

File: pipelines/nextflow/tests/workflows/test_addition_prepare.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    -c ./pipelines/nextflow/workflows/nextflow_test.config \\
+    --cache_dir ./data/test/additional_seq_prepare/cache/ \\
+    --output_dir ./test_addition_seq_prepare_output \\
+    --accession NC_035159.1 \\
+    --prefix TEMP_PREF \\
+    --production_name species_name
+
+  # Check that all the expected files are produced
+  # Make sure to update those if the processing of the files changes!
+  files:
```
Comment (ens-LCampbell) [inline pipelines/nextflow/tests/workflows/test_addition_prepare.yml:30]: Is there any reason why no MD5 checksums are used as in genome_prepare ?

Comment (MatBarba) [reply pipelines/nextflow/tests/workflows/test_addition_prepare.yml:30]: Good catch, I was just happy that it worked just like that, I forgot to add those

File: pipelines/nextflow/tests/workflows/test_genome_prepare.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 # The cached data has been reduced to two seq regions, 1 nuclear and 1 mitochondrial sequence
 - name: genome_prepare_pipeline
   command: nextflow run ./pipelines/nextflow/workflows/genome_prepare/main.nf \\
+    -c ./pipelines/nextflow/workflows/nextflow_test.config \\
```
[the remark this answers is not in the document]
Comment (MatBarba) [reply pipelines/nextflow/tests/workflows/test_genome_prepare.yml:21]: That makes sense, yes

File: pipelines/nextflow/tests/workflows/test_addition_prepare.yml (hunk 1 above)
Comment (MatBarba) [reply pipelines/nextflow/tests/workflows/test_addition_prepare.yml:30]: For now I'll add them, we can always remove/update/ignore those md5 tests as we go

Comment (ens-LCampbell) [reply pipelines/nextflow/tests/workflows/test_addition_prepare.yml:30]: Sounds good thanks Matthieu !

File: pipelines/nextflow/tests/workflows/test_addition_prepare.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test runs the additional_seq_prepare pipeline for the accession NC_035159.1 from Genbank
+# It uses a cache file to avoid downloading the genbank file (--cache_dir)
+# The cached data has been reduced to two genes, 1 protein_coding and 1 tRNA
+- name: genome_prepare_pipeline
```
Comment (Dishalodha) [inline pipelines/nextflow/tests/workflows/test_addition_prepare.yml:19]:
```suggestion
- name: additional_seq_prepare_pipeline
```

**Review by Dishalodha — APPROVED** [review]
Just a change in test pipeline name for additional_seq pipeline, looks great otherwise

**Pull request merged**